skrub/_string_encoder.py

```diff
@@ -89,10 +89,15 @@ def get_feature_names_out(self):
 
         Returns
         -------
-        feature_names_out : list of str objects
+        feature_names_out : ndarray of str objects
```

Comment:
You're outputting a list here
```suggestion
        feature_names_out : list of str objects
```